src/atoms/TextInput/TextInput.native.js

```diff
@@ -43,6 +47,27 @@ const styles = {
         return theme.colors.shade[980];
       }
     },
+    height({ multiline }) {
+      if (!multiline) {
```

Comment:
are you sure about the heights? If you say auto for `multiline` it'll keep on jumping. It should ideally have fixed height and then everything should scroll

Reply:
Also the height for normal textinput doesn't seems like 40px on Figma. Can you verify once?

Reply:
40 was 36+4. 4 for Label to take up some distance on top but now that is changing. 40 will be for outlined & 36 for filled because filled should not have an animating label. It needs to be constant. I'm raising a PR for that, then I'll rebase this and fix these things